src/components/greetings/greeting.js

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
+import { fetchData } from '../redux/greetings/greeting';
+
+function Greeting() {
+  const dispatch = useDispatch();
+  const greetings = useSelector((state) => state.greetings.greeting);
+
+  useEffect(() => {
+    dispatch(fetchData);
+    console.log('hello');
+  }, [dispatch]);
+
+  return (
+    <div>
+      <h3>Hello everyone</h3>
+      {greetings.message}
+    </div>
+  );
```

Comment:
- Hello, Here is what I am getting after connecting rails Api and React. I would advise you to display random greetings and submit them again for review.
-  NB: The servers are running perfectly
![Capture](https://user-images.githubusercontent.com/63972670/215095022-6e7b4041-1a00-4fb9-8d5e-14866d35a562.JPG)